src/bindings/erl_nif/README.md

```diff
@@ -0,0 +1,32 @@
+# Erlang NIF bindings
+
+This binding provides the possibility to access Elektra from Erlang.
+
+## Module name
+
+When compiling the module one needs to set a module name.
+This is because the naming conventions differ between Erlang and Elixir.
+The standard module name `Elixir.Elektra` is for an Elixir binding.
+
+The module name can be set with the CMake option `ELEKTRA_NIF_MODULE_NAME`.
```

Comment:
Is this really needed?

Reply:
Well, if you want to make the binding available for both Erlang and Elixir then you probably would have to provide two separate libraries.

I can think of the following options:

* Provide an option so the user can choose a name (current implementation)
* Using CMake `check_binding_included` allow setting of both `erlang` and `elixir` and compile the library with an appropriate module name.
* Only provide Elixir bindings as this PR only focuses on that

Reply:
Yes, it is better if you concentrate on Elixir for this project. The ideal solution would be if both libraries drop out in one build (like we used to have for e.g. python2 and python3). Please collect such further work in an issue.